hardware_interface/include/hardware_interface/actuator_interface.hpp

```diff
@@ -158,18 +163,23 @@ class ActuatorInterface : public rclcpp_lifecycle::node_interfaces::LifecycleNod
         info_.thread_priority);
       async_handler_->start_thread();
     }
-    return on_init(hardware_info);
+    return on_init(hardware_interface::HardwareComponentInterfaceParams(params));
```

Comment:
not sure about it but if we did not mark the constructor explicit, this could be as short as
```suggestion
    return on_init(params);
```

Reply:
not needed as we are moving away from constructors